packages/@react-aria/landmark/stories/Landmark.stories.tsx

```diff
@@ -0,0 +1,340 @@
+  return (
+    <div>
+      <Navigation aria-label="First Nav">
+        <div>Navigation Landmark with 'First Nav' label</div>
+        <ul>
+          <li><a href="/home">Home</a></li>
+          <li><a href="/about">About</a></li>
+          <li><a href="/contact">Contact</a></li>
+        </ul>
+      </Navigation>
+      <Navigation aria-label="Second Nav">
+        <div>Navigation Landmark with 'Second Nav' label</div>
+        <ul>
+          <li><a href="/home">Home</a></li>
+          <li><a href="/about">About</a></li>
+          <li><a href="/contact">Contact</a></li>
+        </ul>
+      </Navigation>
+      <Main>
+        <div>Main Landmark</div>
+        <TextField label="First Name" />
+      </Main>
+    </div>
+  );
+}
+
+function DuplicateRolesWithSameLabelsExample() {
+  return (
+    <div>
+      <Navigation aria-label="First Nav">
+        <div>Navigation Landmark with 'First Nav' label</div>
+        <ul>
+          <li><a href="/home">Home</a></li>
+          <li><a href="/about">About</a></li>
+          <li><a href="/contact">Contact</a></li>
+        </ul>
+      </Navigation>
+      <Navigation aria-label="First Nav">
+        <div>Navigation Landmark with 'First Nav' label</div>
+        <ul>
+          <li><a href="/home">Home</a></li>
+          <li><a href="/about">About</a></li>
+          <li><a href="/contact">Contact</a></li>
+        </ul>
+      </Navigation>
+      <Main>
+        <div>Main Landmark</div>
+        <TextField label="First Name" />
+      </Main>
+    </div>
+  );
+}
+
+function DuplicateRolesNoLabelExample() {
+  return (
+    <div>
+      <Navigation>
+        <div>Navigation Landmark with no label</div>
+        <ul>
+          <li><a href="/home">Home</a></li>
+          <li><a href="/about">About</a></li>
+          <li><a href="/contact">Contact</a></li>
+        </ul>
+      </Navigation>
+      <Navigation>
+        <div>Navigation Landmark with no label</div>
+        <ul>
+          <li><a href="/home">Home</a></li>
+          <li><a href="/about">About</a></li>
+          <li><a href="/contact">Contact</a></li>
+        </ul>
+      </Navigation>
+      <Main>
+        <div>Main Landmark</div>
+        <TextField label="First Name" />
+      </Main>
+    </div>
+  );
+}
+
+function OneWithNoFocusableChildrenExample() {
+  return (
+    <div>
+      <TextField label="First Name" />
+      <Main>
+        <div>Main Landmark</div>
+        <div>No focusable children</div>
+      </Main>
+      <TextField label="First Name" />
+    </div>
+  );
+}
+
+function AllWithNoFocusableChildrenExample() {
+  return (
+    <div>
+      <Region>
+        <div>Region Landmark</div>
+        <div>No focusable children</div>
+      </Region>
+      <TextField label="First Name" />
+      <Main>
+        <div>Main Landmark</div>
+        <div>No focusable children</div>
+      </Main>
+    </div>
+  );
+}
+
+function NestedExample() {
+  return (
+    <div>
+      <Main>
```

Comment:
Should this `main` have an `aria-labelledby` pointing to the `Main Landmark` div so it gets announced when focused via the f6 key? Doesn't seem to announce at the moment, but then again the WAI-ARIA stuff I looked at seems to indicate you don't need a label if it is the only `main` on the page? Other landmark roles seem to have varying rules on if they should be labeled:  https://www.w3.org/TR/wai-aria-practices/examples/landmarks/region.html

Reply:
Also are we supposed to be able to move focus back onto the landmark itself once focus is moved to a focusable child within. Curious if the landmark label announcement will happen if focus lands inside the landmark rather than on the landmark

Reply:
1. I can just make the `Main Landmark` and similar more semantic with `h1`, `h2`, etc. That should make it announce how we want. I was just wanting a visual indicator to make testing in the storybook easier.
2. Currently, we don't move focus back to the landmark itself if previously focused within that landmark (as outlined in https://github.com/adobe/react-spectrum/pull/2750#discussion_r797903401). Definitely open to changing this, but I'm not sure which would be better.
3. In VO it seems like it announces the parent landmark for some things (text inputs), but not others (links). Try the first story to see the difference.

Reply:
1. I'm fine with either since this is a story level thing, no need for it to be a blocker. 
2. Gotcha, thanks for the comment reference. Happy for this to be handled later as per the comment
3. Hm, ok seems to be a case by case thing then